src/workers/postScoutMatchedMail.ts

```diff
@@ -0,0 +1,56 @@
+import { templateId } from '../common/mailing';
+import { messageToJson, Worker } from './worker';
+import { Post } from '../entity';
+import {
+  fetchUser,
+  formatPostCreatedAt,
+  getDiscussionLink,
+  pickImageUrl,
+} from '../common';
+import {
+  baseNotificationEmailData,
+  sendEmail,
+  truncatePostToTweet,
+} from '../common';
+
+interface Data {
+  postId: string;
+  scoutId: string;
+}
+
+// TODO:: Update the dynamic template data once available
+
+const worker: Worker = {
```

Comment:
We need to send the email only when the post is approved and added to the DB not when it is matched

Reply:
This gets triggered when a new post is created in the DB. Do you want it to be moved when the submission status has changed to approved?

https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/190939137/WT-4+Community+Links#API%2FGateway-changes

Reply:
Nope. My bad